lib/ansible/module_utils/facts/system/systemd.py

```diff
@@ -39,9 +39,17 @@ def collect(self, module=None, collected_facts=None):
             if rc != 0:
                 return systemd_facts
 
-            systemd_facts["systemd"] = {
-                "features": str(stdout.split("\n")[1]),
-                "version": int(stdout.split(" ")[1]),
-            }
+            systemd_version = str(stdout.split(" ")[1])
+
+            systemd_facts["systemd"] = {}
+            systemd_facts["systemd"]["features"] = str(stdout.split("\n")[1])
```

Comment:
`str.split()` already returns a list of strings. No need to convert.
```suggestion
            systemd_facts["systemd"]["features"] = stdout.split("\n")[1]
```